src/stitch/blender.cc

```diff
@@ -42,7 +42,7 @@ Mat32f LinearBlender::run() {
 		memset(weight.ptr(), 0, target_size.y * target_size.x * sizeof(float));
 		fill(target, Color::BLACK);
 #pragma omp parallel for schedule(dynamic)
-		REP(k, images.size()) {
+		REP(k, (int)images.size()) {
```

Comment:
Not the problem of size_t, but that the iteration related to  Openmp on windows need int.